src/app/clients/clients-view/view-share-account/view-share-account.component.html

```diff
@@ -0,0 +1,151 @@
+<mat-card>
+<mat-card-title>
+      <div class="share-account-header">
+        <h4>
+          <i class="fa fa-stop" [ngClass]="shareAccount?.status?.code|statusLookup"></i>
+          {{shareAccount.productName}}({{shareAccount.accountNo}})
+        </h4>
+        <button mat-raised-button color="primary"><i class="fa fa-pencil front-icons"></i>Modify Application</button>
+        <button mat-raised-button color="primary"><i class="fa fa-check front-icons"></i>Approve</button>
+        <button mat-raised-button color="primary" [matMenuTriggerFor]="More">More<i class="fa fa-caret-down rear-icons"></i></button>
+        <mat-menu #More="matMenu">
+          <button mat-menu-item>Reject</button>
+          <button mat-menu-item>Delete</button>
+        </mat-menu>
+      </div>
+</mat-card-title>
+<mat-card-content>
+  <div class="share-account-tables">
+    <table>
+      <tbody>
+        <tr>
+          <td >Activated On</td>
+              <td>
+                <span *ngIf = "shareAccount.status.value === 'Active'">{{shareAccount.timeline.activatedDate | date}}</span>
+                <span *ngIf = "shareAccount.status.value !== 'Active'">Not Activated</span>
+              </td>
+        </tr>
+        <tr>
+          <td >Currency</td>
+          <td >{{shareAccount.currency.name}}</td>
+        </tr>
+        <tr>
+          <td >External Id</td>
+          <td></td>
+          <!-- <td >{{shareAccount.externalId}}</td> -->
```

Comment:
why is this commented?

Reply:
External Id field is yet to be added to the response for retrieve share account request. I have modified it to `shareAccount?.externalId` now so that it automatically reflects the required data once added.